internal/pkg/pluginengine/cmd_apply.go

```diff
@@ -23,7 +23,7 @@ func Apply(configFile string, continueDirectly bool) error {
 
 	err = pluginmanager.CheckLocalPlugins(cfg)
 	if err != nil {
-		log.Errorf("Error checking required plugins. Maybe you forgot to run \"dtm init\" first?")
+		log.Error("Error checking required plugins. Maybe you forgot to run \"dtm init\" first?")
```

Comment:
Please also change line 21 for the same.

Reply:
Since the fmt.Error doesn't exist so I have to change it either to log.Error or fmt.Print 
that might loose the lable of [Info] if I go for fmt.Print @daniel-hutao please into the same.

Reply:
`fmt.Errorf` don't need to change for the `fmt.Error` doesn't exist. 

https://github.com/devstream-io/devstream/issues/537#issuecomment-1131061143